client/lib/domains/whois/test/assembler.js

```diff
@@ -6,105 +6,144 @@ const expect = require( 'chai' ).expect;
 /**
  * Internal dependencies
  */
-const whoisAssembler = require( './../assembler' );
+const whoisAssembler = require( './../assembler' ),
+	{ whoisType } = require( './../constants' );
+
+const FIRST_NAME = 'First-name',
+	LAST_NAME = 'Last-name',
```

Comment:
Coming from C++ and Java, I also prefer this style and noticed the same trend as you - but until it's officially in the guidelines, I'd prefer not to start converting old code to it.